payments-core/src/main/java/com/stripe/android/model/DelicateStripeApi.kt

```diff
@@ -0,0 +1,9 @@
+package com.stripe.android.model
+
+@RequiresOptIn(
+    level = RequiresOptIn.Level.ERROR,
+    message = "This is a delicate API that is only intended for advanced users and not required " +
+        "for most integrations.",
```

Comment:
Should we have more specific wording here, or maybe even a different annotation name?

Reply:
I think we should have a different annotation name, something like `@PCIComplianceRequired` or something, to let integrators know that they may be breaking PCI compliance.